src/cephadm/cephadm

```diff
@@ -3241,10 +3241,11 @@ def command_adopt_prometheus(daemon_id, fsid):
     config_src = '/etc/prometheus/prometheus.yml'
     config_src = os.path.abspath(args.legacy_dir + config_src)
     config_dst = os.path.join(data_dir_dst, 'etc/prometheus')
+    makedirs(config_dst, uid, gid, 0o755)
     copy_files([config_src], config_dst, uid=uid, gid=gid)
 
     # data
-    data_src = '/var/lib/prometheus/metrics/'
```

Comment:
This path appears to be platform and/or config dependent based the `--storage.tsdb.path` argument.

Debian uses a default of `/var/lib/prometheus/metrics2/`, whereas SUSE uses a default of `/var/lib/prometheus/metrics/` ...

Reply:
Right, so we should have something in cephadm to be able to override the prometheus data directory because it depends on the OS.

I'll remove that part of this PR and only address the makedirs issue

@sebastian-philipp as I'll remove the change on the data directory, I won't need to modify cephadm-adoption-corpus